plutus-core/untyped-plutus-core/test/Evaluation/Builtins/Conversion.hs

```diff
@@ -47,7 +47,7 @@ i2bProperty1 = do
   e <- forAllWith ppShow Gen.bool
   -- We limit this temporarily due to the limit imposed on lengths for the
   -- conversion primitive.
-  d <- forAllWith ppShow $ Gen.integral (Range.constant 0 integerToByteStringMaximumOutputLength)
+  d <- forAllWith ppShow $ Gen.integral (Range.constant 0 Bitwise.maximumOutputLength)
```

Comment:
I wonder if we should be using the fancy `QuickCheck` constant generator in some of these tests.